scripts/release/mule/Makefile.mule

```diff
@@ -46,32 +46,30 @@ ci-build: ci-setup buildsrc gen
 		 NO_BUILD=True VARIATIONS=$(OS_TYPE)/$(ARCH) \
 		 scripts/build_packages.sh $(OS_TYPE)/$(ARCH)
 
-package-$(OS_TYPE)-$(ARCH): package-deb-$(OS_TYPE)-$(ARCH) package-rpm-$(OS_TYPE)-$(ARCH)
-
-package-deb-$(OS_TYPE)-$(ARCH):
+package-deb:
 	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/deb/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
 
-package-rpm-$(OS_TYPE)-$(ARCH):
-	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/rpm/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
-
-package-docker-$(OS_TYPE)-$(ARCH):
+package-docker:
 	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/docker/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
 
-package-test-$(OS_TYPE)-$(ARCH): package-test-deb-$(OS_TYPE)-$(ARCH) package-test-rpm-$(OS_TYPE)-$(ARCH)
+package-rpm:
+	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/rpm/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
 
-package-test-deb-$(OS_TYPE)-$(ARCH):
-	scripts/release/mule/test/$(OS_TYPE)/$(ARCH)/test.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) deb
+package-sign-deb:
+	scripts/release/mule/sign/$(OS_TYPE)/$(ARCH)/sign.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) deb
 
-package-test-rpm-$(OS_TYPE)-$(ARCH):
-	scripts/release/mule/test/$(OS_TYPE)/$(ARCH)/test.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) rpm
+package-sign-rpm:
+	scripts/release/mule/sign/$(OS_TYPE)/$(ARCH)/sign.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) rpm
+
+package-sign-tarball:
+	scripts/release/mule/sign/$(OS_TYPE)/$(ARCH)/sign.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) tar.gz
 
-package: package-$(OS_TYPE)-$(ARCH)
-package-deb: package-deb-$(OS_TYPE)-$(ARCH)
-package-rpm: package-rpm-$(OS_TYPE)-$(ARCH)
+package-sign-source:
+	scripts/release/mule/sign/$(OS_TYPE)/$(ARCH)/sign.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) source
 
-package-docker: package-docker-$(OS_TYPE)-$(ARCH)
+package-test-deb:
+	scripts/release/mule/test/$(OS_TYPE)/$(ARCH)/test.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) deb
 
-package-test: package-test-$(OS_TYPE)-$(ARCH)
-package-test-deb: package-test-deb-$(OS_TYPE)-$(ARCH)
-package-test-rpm: package-test-rpm-$(OS_TYPE)-$(ARCH)
+package-test-rpm:
+	scripts/release/mule/test/$(OS_TYPE)/$(ARCH)/test.sh $$WORKDIR $(OS_TYPE) $(ARCH) $(ARCH_BIT) $(VER) rpm
 
```

Comment:
These targets are way too verbose, however, I have a commit that will simplify them and will create a PR when this code has been merged.